plugins/modules/sefcontext.py

```diff
@@ -17,11 +17,14 @@
   - Similar to the C(semanage fcontext) command.
 extends_documentation_fragment:
   - community.general.attributes
+  - community.general.attributes.platform
 attributes:
   check_mode:
     support: full
   diff_mode:
     support: full
+  platform:
+    platforms: rhel
```

Comment:
Hmm, so far the documentation never stated that this module is RHEL only. How about all other Linuxes that allow to use selinux?

Reply:
Which distributions other than RHEL-based distros support SELinux? I guess SUSE, something else? I've only seen "rhel", "debian", "posix" in modules should it be changed to `rhel, suse`?

Reply:
Essentially all of them I know. Debian, Ubuntu, Alpine, Arch Linux, ... Check for example https://github.com/SELinuxProject/selinux#installation

Reply:
OK. Last time I looked into APT distros they were AppArmor only and shunned SELinux, I guess things have improved since. I've updated it to `platforms: linux`.
Thanks.

Reply:
Debian seem to have supported selinux since at least 2004 (https://wiki.debian.org/SELinux?action=recall&rev=1). Anyway, `platforms: linux` sounds good.